src/main/java/teammates/common/util/TeammatesDateTimeFormatter.java

```diff
@@ -0,0 +1,21 @@
+package teammates.common.util;
+
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
+
+/**
+ * DateTimeFormatter for TEAMMATES with default Locale.US settings.
+ */
+public final class TeammatesDateTimeFormatter {
+
+    private TeammatesDateTimeFormatter() {
+        // Utility class
+    }
+
+    /**
+     * Returns DateTimeFormatter instance with Locale.US as default locale.
+     */
+    public static DateTimeFormatter ofPattern(String pattern) {
```

Comment:
Return a `TeammatesDateTimeFormatter` instead.

Also, add a `format` method that we can use instead.

Reply:
> Return a TeammatesDateTimeFormatter instead.


How exactly?

My idea is that since we set the locale while setting the pattern, I will only wrap that function so that after that function returns the `DateTimeFormatter` we can use other DateTimeFormatter functions with the required set locale. 
This will save us from the hassle of wrapping every function and will make sure code works once TeammatesDateTimeFormatter.ofPattern has been used

Similarly for this https://github.com/TEAMMATES/teammates/pull/8977/files#r204227822